pkg/jobs/status.go

```diff
@@ -54,7 +54,7 @@ func StartStatusReporter(ctx context.Context, scan_id string, scanner *scan.Scan
 		}
 
 		if scanner.Stopped.Load() == true {
-			output.WriteScanStatus("ERROR", scan_id, "Scan stopped by user")
+			output.WriteScanStatus("CANCELLED", scan_id, "Scan stopped by user")
```

Comment:
nit: Ideally we should feed from `deepfence_utils` where `SCAN_STATUS_CANCELLED` is defined

Reply:
Let me change that in a separate PR.